README.md

```diff
@@ -17,4 +17,169 @@
+
+```js
+const simba = {
+  'animal type': 'lion'
+};
+```
+
+To print this property, we'd have to use the bracket notation instead of the dot notation. This is because if we were to type `console.log(simba.animal type);`, it would get interpreted in the wrong way. We need to nudge JS in the right direction:
+
+```js
+console.log(simba['animal type']); // prints 'lion'
+```
+
+You might notice that this looks a little verbose. You will pretty much never use the bracket notation like this with an inline string. The bracket notation does have one neat trick up its sleeve, though. You can use a variable inbetween the brackets, and the variable's value will get used as a key. In this example, that would be the `'species'` string:
+
+```js
+const propertyName = 'species';
+console.log(simba[propertyName]); // prints 'lion'
+```
+
+Neat! We'll see why this is useful in just a minute, when we get to iteration. But first things first...
+
+
+## We need more data!
+Remember how we recreated our object earlier on in the lesson, when we wanted to add the `species` property to that empty object we created before? We don't have to do that. We can easily add properties to objects, just like we would assign variables:
+
+```js
+simba.furColor = 'golden';
+simba.sleep = function () {
+  console.log('Zzz.');
+};
+```
+
+We can also do this using the bracket notation:
+
+```js
+simba['height'] = 120;
+```
+
+## Fat cat
+Simba's a pretty heavy cat for his age, and he's fairly self-conscious about it. He'd much prefer it if we left his weight out of our data. There's one problem though: it's already in there! To remove a property from an object, we use the `delete` keyword:
+
+```js
+console.log(simba.weight); // prints 190
+delete simba.weight;
+console.log(simba.weight); // prints undefined
+```
+
+## Looping the loop
+To cap things off, we'll print out a brief summary of all the data points we have on our furry friend. A naive implementation would look something like this:
+
+```js
+console.log(simba.species); // prints 'lion'
+console.log(simba.name); // prints 'Simba'
+console.log(simba.weight); // prints undefined
+console.log(simba.age); // prints 8
+console.log(simba.diet); // prints ['antelope', 'tortoise', 'zebra']
+console.log(simba.personality); // prints Object {friendly: true, aggressive: false}
+console.log(simba.roar); // prints function () { ... }
+```
+
+This, however, is super inefficient. We deleted the weight, so we're not interested in that property anymore. And what about the other properties we've added during the course of the lesson? Doing this manually sucks! Luckily, there's an easy way to iterate over our object's properties and log them out, using the key and the value:
+
+```js
+for (const key in simba) {
+  if (simba.hasOwnProperty(key)) {
```

Comment:
I would omit `hasOwnProperty()` for now, as the current example will work just fine without.

Omitting it here has the virtue of allowing us, later on, to introduce students to object inheritance where we loop through an object _without using_ `hasOwnProperty()` and see the properties that we wouldn't have expected. Then we can introduce `hasOwnProperty()` and really drive home how it works and why it's important.

Seeing the error first makes it easier to remember the solution.